### DISCUSSION TITLE
MY GANACHE SOFTWARE CAN'T SHOW THE MAIN TAB (TAB THAT SHOWS ADRESSES) [WIN 10]

### DISCUSSION THREAD

**Discussion by Qyuzet** [ConsenSys-archive/truffle#5572] [General Q&A]
Hi, I have some trouble with my Ganache software on  my laptop, i run this on Windows 10, the tab is stuck on the settings tab after clicking the "QUICKSTART ETHEREUM" button option, and can't get into the main tab that shows addresses and else...., I've tried to re-install the app but still the same, I also tried to click the "NEW WORKSPACE" option, but the app screen was only showing the white blank, my Ganache app was working well previously, can anybody know how to fix it?

![image](https://user-images.githubusercontent.com/93258081/192805359-69580a5b-dcea-4dc3-a954-2cd446aeecd6.png)

I just want to go to this tab as like as normal when i start the "QUICKSTART ETHEREUM" :

![ganache-window](https://user-images.githubusercontent.com/93258081/192808406-01403bea-ecce-4959-b0e6-b82195baf3dd.png)

**Comment by apdan510** [comment]
Hey @Qyuzet Can you please try uninstalling your Ganache-ui, restarting your computer and reinstalling this version of Ganache https://github.com/trufflesuite/ganache-ui/releases/download/v2.5.4/Ganache-2.5.4-win-setup.exe
